dwl.c

```diff
@@ -780,6 +787,10 @@ commitnotify(struct wl_listener *listener, void *data)
 	/* mark a pending resize as completed */
 	if (c->resize && c->resize <= c->surface.xdg->configure_serial)
 		c->resize = 0;
+
+	// Damage the whole screen
+	if (c->mon)
+		wlr_output_damage_add_whole(c->mon->damage);
```

Comment:
Can `c->mon` ever be null if we get here?

Reply:
This solved an out of bounds panic I was encountering. If we can guarantee that `commit` is never called before a surface is mapped, then this wouldn't be necessary, right?

Reply:
Ah, you're right, the surface can be committed before attaching a buffer (which would cause it to be mapped on the following commit).  This makes sense then... and now, having thought it through, seems like adding `&& VISIBLEON(c, c->mon)` would be even better.